SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -351,6 +355,46 @@ bool FLocalDeploymentManager::SetupRuntimeFileLogger(const FString& RuntimeLogDi
 }
 
 bool FLocalDeploymentManager::TryStopLocalDeployment()
+{
+	if (!StartLocalDeploymentShutdown())
```

Comment:
How does this look from the editor while it's stopping now? Could you record and upload a gif please